lib/ui/painting.dart

```diff
@@ -1568,6 +1568,20 @@ class Paint {
   }
 }
 
+/// The color space in which all of an [Image]'s colors exist.
+///
+/// This value can help you decide which [ImageByteFormat] to use with
+/// [Image.toByteData] and is the result of [Image.colorSpace].
+enum ColorSpace {
+  /// The sRGB color space, the defined standard color space for the web.
```

Comment:
> the defined standard color space for the web.

This is kind of an odd statement, I'm not sure how that helps me to know what it means for flutter. Perhaps describe this in terms of the values the colors can take and/or what encoding to use?

Reply:
I don't think this value has any effect on what encoding you should use.  The only way I could describe the values would be to list the frequencies in the gamut.  Did you have a different idea on how we could specify it?  It is kind of weird to list the web standard but it's the only thing I could grasp onto where people would have an existing familiarity with sRGB.

Reply:
defined by whom? linking to the relevant standard would be useful for people new to this topic

Reply:
Citations are provided in the wiki (but it was standardized by the International Electrotechnical Commission (IEC) as IEC 61966-2-1).  Linking the wikipedia page.

Reply:
The important part isn't whether or not this is the defined color space for the web. As a flutter developer, what does the web's color space have to do with my application? If we don't have a good answer, maybe just something like "this is the default color space for applications" or something like that

Reply:
Okay, reworded it and added note about it being the colorspace of non-wide-gamut apps.